extension/packages/hardhat/hardhat.config.ts.args.mjs

```diff
@@ -0,0 +1,12 @@
+export const compilers = [
+  {
+    version: "0.8.20",
+    settings: {
+      optimizer: {
+        enabled: false,
```

Comment:
I had to disable the optimizer to avoid getting a stack-too-deep error. Another option is to use viaIR.

Reply:
> Another option is to use viaIR.

What is it?

Reply:
It's another way to compile Solidity https://soliditylang.org/blog/2024/07/12/a-closer-look-at-via-ir/